cryptoki/src/error/mod.rs

```diff
@@ -18,7 +20,7 @@ pub enum Error {
     LibraryLoading(libloading::Error),
 
     /// All PKCS#11 functions that return non-zero translate to this error.
-    Pkcs11(RvError),
+    Pkcs11(RvError, Function),
```

Comment:
This looks like an API break...? :thinking: (nothing against it just noting for version bump).

Reply:
Hmm, indeed. I was aware, but now thinking about it, maybe I can hide the `Function` in `RvError` instead. Will need to investigate